libs/ml/include/ml/model/sequential.hpp

```diff
@@ -45,12 +45,14 @@ class Sequential : public Model<TensorType>
   template <typename LayerType, typename... Params>
   void Add(Params... params);
 
+  SizeType LayerCount() const;
+
   template <typename X, typename D>
   friend struct serializers::MapSerializer;
 
 private:
-  SizeType    layer_count_ = 0;
   std::string prev_layer_;
+  SizeType    layer_count_ = 0;
```

Comment:
if you add any new member variables to an object, this will need to be reflected in the object serialisation and deserialistion. Please speak to me, Jiri, or Emma if you're not sure exactly how to do that.

Reply:
Ok, but this variable actually already exists in
https://github.com/fetchai/ledger/blob/master/libs/ml/include/ml/model/sequential.hpp
and there are several lines of code using it.
I don't know why Github highlighted this line as a change, becase I've added a getter method, not a member.

Reply:
Serialisation/deserialisation of this member is already implemented in sequential.hpp, so no action required.